spec/features/citation_spec.rb

```diff
@@ -3,14 +3,20 @@
 require 'rails_helper'
 
 describe 'citation' do
-  let(:bibid) { '7994866' }
-
+  before do
```

Comment:
Question: Why extract this to a before? Are there other tests depending on this?

Reply:
No there are not for now. I can move it in the it.